internal/awsservice/cloudwatchlogs.go

```diff
@@ -97,7 +96,7 @@ func getLogsSince(logGroup, logStream string, since, until *time.Time) ([]string
 		if err != nil {
 			if errors.As(err, &rnf) && attempts <= StandardRetries {
 				// The log group/stream hasn't been created yet, so wait and retry
-				time.Sleep(time.Minute)
+				time.Sleep(30 * time.Second)
```

Comment:
Reduce the number of time for validating logs. Right now, for every time we call `ValidateLogs`, if we fail, we retry for 1 minute with a total of 5 retries ( 5 minutes); therefore, reducing to 30s with a total of 3 retries (1'30s)